### ISSUE TITLE
AdMob docs are messy and incorrect

### ISSUE THREAD

**Issue by rodrigograca31** [ionic-team/ionic-docs#698]
**URL**
(URLs bellow)

**What is missing or inaccurate about the content on this page?**
The current docs as well as the V3 docs have multiple admob pages some invalid and 1 is from a plugin that[ steals money from developers (`cordova-plugin-admob` or `cordova-plugin-admobpro`)](https://github.com/ratson/cordova-plugin-admob-free/issues/354)

I propose you remove this pages (and the associated modules code):
**V4 docs**
https://ionicframework.com/docs/native/admob (non existent)
https://ionicframework.com/docs/native/admob-pro (money stealing plugin) (module: @ionic-native/admob-pro/ngx)
**V3 docs:**
https://ionicframework.com/docs/v3/native/admob-pro/ (module: @ionic-native/admob-pro)
"AdMob" and "Admob" is repeated and redirects to "AdMob-Pro" (see image bellow)
![image](https://user-images.githubusercontent.com/1134310/57851957-c9dfb280-77d9-11e9-9182-d82f2a7e3b6f.png)

**Comment by mhartington** [comment]
### V4

The `https://ionicframework.com/docs/native/admob` link links to a valid plugin. Not sure what you mean 

As for AdMob-Pro, I wouldn't really say that this is "stealing" money, as they clearly state that it is a commercial plugin. I think we could modify the docs to add back the "Paid" flag in the menu and make the note that it is a paid plugin a bit clearer. 

### V3

You are correct here! We'll get this cleaned up. 


Thanks for the issue 😄

**Comment by rodrigograca31** [comment]
> The `https://ionicframework.com/docs/native/admob` link links to a valid plugin. Not sure what you mean

nope, it redirects to one, the exact link doesnt exist, if you click on "Admob" on the menu you get a "uh oh" page.

Yes that plugin steals from developers. You can [see my full documentation here](https://github.com/ratson/cordova-plugin-admob-free/issues/354) but in short: if I tell you: hey you got to pay a $20 licence or I get 2% of your revenue and then I'm secretly controlling your ads with my server side code and actually taking 30% without ever telling you or disclosing that anywhere or putting it in the license, and trying to hide that by disabling issues, isnt that stealing? (taking 30% without prier agreement!)

**Comment by rodrigograca31** [comment]
You might not call it stealing but if the ionic team wants to promote a plugin with server side control over ads that's up to you guys.
